vllm/model_executor/models/bert.py

```diff
@@ -387,10 +387,18 @@ def __init__(
         config: BertConfig,
         cache_config: Optional[CacheConfig] = None,
         quant_config: Optional[QuantizationConfig] = None,
+        pooler_config: Optional[PoolerConfig] = None,
     ) -> None:
         super().__init__()
         self.model = BertModel(config, cache_config, quant_config)
-        self._pooler = Pooler(pooling_type=PoolingType.CLS, normalize=True)
+        self._pooler = Pooler(
+            pooling_type=PoolingType[pooler_config.pooling_type]
+            if pooler_config.pooling_type is not None else PoolingType.CLS,
+            normalize=pooler_config.pooling_norm or True,
+            softmax=pooler_config.pooling_softmax or False,
+            step_tag_id=pooler_config.pooling_step_tag_id,
+            returned_token_ids=pooler_config.pooling_returned_token_ids,
+        )
```

Comment:
Can we add a factory method to `Pooler` so automatically merge the config with model specific defaults?

Reply:
e.g. we should be able to write

```
self._pooler = Pooler.from_config_with_defaults(
    pooler_config,
    default_pooling_type=PoolingType.CLS,
    default_normalize=True,
)
```

Reply:
Please check it